WebDevStudios/wds-blocks
# Add documentation to global `utils` directory

Addresses #88

### Description

- Moves, renames, updates references, and adds `README.md` files for global `components` and `hooks` not already addressed as part of other tickets:
    - `PreviewToggle`
    - `withBackgroundColor`
    - `withBackgroundImage`
    - `withBackgroundVideo`
    - `withFontColor`
    - `usePreviewToggle`

### Steps to verify the solution

No functional changes. Add/edit Accordion and Carousel blocks to confirm no errors/issues encountered following `utils` reorganization.

### Other

- [x] Is this issue accessible? (Section 508/WCAG 2.1AA)
- [ ] Does this pass CrossBrowserTesting.com? (Edge, Safari, Chrome, Firefox)
- [ ] Will this pull request require [updating the wiki?](https://github.com/WebDevStudios/wds-block-starter/wiki)
